src/test/java/com/pngencoder/PngEncoderBenchmarkAssorted.java

```diff
@@ -40,25 +42,32 @@ public void runBenchmarkCustom() throws IOException {
 
         final File outImageIO = File.createTempFile("out-imageio", ".png");
         //final File outPngEncoder = File.createTempFile("out-pngencoder", ".png");
-        final File outPngEncoder = new File("/Users/olof/Desktop/out.png");
+        final File outPngEncoder = new File("target/test/assorted_out.png");
```

Comment:
`olof`. Nice. I guess whatever is better than that :)
I'm not sure why we have two kinds of benchmarks. There is also https://github.com/pngencoder/pngencoder/blob/develop/src/test/java/com/pngencoder/PngEncoderBenchmarkPngEncoderVsImageIO.java, which has built in warmup and aggregation of results.